types/gensync/index.d.ts

```diff
@@ -0,0 +1,149 @@
+// Type definitions for gensync 1.0
+// Project: https://github.com/loganfsmyth/gensync
+// Definitions by: Jake Bailey <https://github.com/jakebailey>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.0
+
+import { GensyncGenerator, Operation, Options } from '.';
+
```

Comment:
```suggestion
```

Reply:
I can change it to that, but I can't say I'm super happy with the redundancy in the tooltips.